tck/src/main/java/com/hedera/hashgraph/tck/methods/sdk/param/CommonTransactionParams.java

```diff
@@ -0,0 +1,59 @@
+package com.hedera.hashgraph.tck.methods.sdk.param;
+
+import com.hedera.hashgraph.sdk.Client;
+import com.hedera.hashgraph.sdk.Hbar;
+import com.hedera.hashgraph.sdk.PrivateKey;
+import com.hedera.hashgraph.sdk.Transaction;
+import java.time.Duration;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+/**
+ * CommonTransactionParams
+ */
+@SuppressWarnings("OptionalUsedAsFieldOrParameterType")
+@Getter
+@AllArgsConstructor
+@NoArgsConstructor
+public class CommonTransactionParams {
```

Comment:
There is no `transactionId` as per [spec](https://github.com/hashgraph/hedera-sdk-tck/blob/main/test-specifications/commonTransactionParameters.md#transaction-parameter-object-definition)

Reply:
added